.github/workflows/ci.yml

```diff
@@ -47,28 +47,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8]
+        os: [ubuntu-latest, windows-latest, macos-latest, macos-11.0]
+        python-version: [3.7, 3.8, 3.9]
         browser: [chromium, firefox, webkit]
-        include:
```

Comment:
This was intended and by your change it increases the jobs from 26 to 38 which is I think not needed. It's already checked with Python 3.9.

Reply:
@mxschmitt I changed it as I assumed as this project is open source so it must be on a free tier. Anyway if you think so I will change it